src/strategies/dgenesis/examples.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "name": "dGenesis Query",
+    "strategy": {
+      "name": "dgenesis",
+      "params": {
+        "address": "0xbe1fef07fb53aed268d8a7b1d6eff0c18c09db28",
```

Comment:
This is not used anywhere? we can remove it? also would be great if you can add a readme file to help others understand what this strategy is about

Reply:
Removed the address line and added readme in updated commit.